RoaringBitmap/src/test/java/org/roaringbitmap/TestIterators.java

```diff
@@ -207,4 +208,85 @@ public void testEmptySkips() {
     PeekableIntIterator it = bitmap.getIntIterator();
     it.advanceIfNeeded(0);
   }
+
+  @Test
+  public void testSkipIntoGaps() {
+    RoaringBitmap bitset = new RoaringBitmap();
```

Comment:
More tests are always welcome, of course, but is this just a sanity check to verify the same bug wasn't also in the 32 bit implementation?

Reply:
Yes, I wanted to make sure it's not a problem with the containers, but with the ART.

Reply:
This sounds like a good idea.